packages/terra-list/src/List.scss

```diff
@@ -1,16 +1,62 @@
-@import '~terra-mixins';
 @import './variables';
 
-.terra-List {
-  list-style: none;
-  margin: 0;
-  padding: 0;
-}
+// List Component
+// ========================================================
+/* stylelint-disable selector-class-pattern, selector-max-compound-selectors */
+:local {
+  .list {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  .divided > .list-item {
+    border-top: solid 1px $terra-list-divider-color;
```

Comment:
I'm not sure what's the conventions to follow. By following the code base, [link](https://github.com/cerner/terra-core/blob/8a8218478eb78934ee469d2d9b71d7d63a4b5d24/packages/terra-table/src/_variables.scss#L16-L19).
I will update it to:
```scss
// _variables.scss
$terra-list-border-width: 1px;
$terra-list-border-top: solid $terra-list-border-width $terra-list-divider-color;
$terra-list-border-bottom: solid $terra-list-border-width $terra-list-divider-color;
```
In list.scss file:
```scss
.divided > .list-item {
    border-top: $terra-list-border-top;

    &:last-of-type {
      border-bottom: $terra-list-border-bottom;
    }
  }
```
Does this work?